HWLesson2/Task5.js

```diff
@@ -0,0 +1,18 @@
+const checkForSpam = function(message) {
```

Comment:
Можна дещо спростити функцію і не створювати змінну :) 
```suggestion
const checkForSpam = function(message) {
   if (message.toLowerCase().includes('spam')) {
     return true;
   }
  
  // dafault value
  return false;
}
```